llvm/lib/CodeGen/SelectionDAG/InstrEmitter.cpp

```diff
@@ -1412,6 +1412,13 @@ EmitSpecialNode(SDNode *Node, bool IsClone, bool IsCloned,
       }
     }
 
+    // Add rounding control registers as implicit def for inline asm.
+    if (MF->getFunction().hasFnAttribute(Attribute::StrictFP)) {
```

Comment:
GlobalIsel needs equivalent

Reply:
Done. But I cannot add a test for it. IIUC, X86 GlobalIsel haven't supported strict FP.

Reply:
Doesn't need x86

Reply:
Yes, I'm just familiar with none of them 😅